### PR TITLE
[v0.3] Refine PyPI nightly build release version issue

### PR DESCRIPTION
**Pull request by zigzagcai**
1. Refine e2eAIOK PyPI nightly build release version

-          automatic increment minor version number for nightly:
-          such as 0.2.4 (stable release)--> 0.2.5b20221206 (pre-release nightly)

### CODE REVIEW

**Comment by zigzagcai** [comment]
Therefore, user can install pre-release nightly PyPI package by typing 
`python -m pip install e2eAIOK --pre`

**Comment by zigzagcai** [comment]
We are following best practices from BigDL. The pre-release nightly version is one minor number ahead of stable release.

File: .github/workflows/e2eaiok_nightly_pypi.yml [hunk 1]
```diff
@@ -34,8 +34,9 @@ jobs:
       run: |
         pip install build wheel
         release_version=$(cat e2eAIOK/version | head -1)
-        nightly_build_date=`date '+%Y%m%d%H'`
-        nightly_version=${release_version}b${nightly_build_date}
+        next_release_version=$(echo ${release_version} | awk -F. -v OFS=. 'NF==1{print ++$NF}; NF>1{$NF=sprintf("%0*d", length($NF), ($NF+1)); print}')
+        nightly_build_date=`date '+%Y%m%d'`
```
[the remark this answers is not in the document]
Comment (zigzagcai) [reply .github/workflows/e2eaiok_nightly_pypi.yml:38]: Thanks! I will fix it.

[the remark this answers is not in the document]
Comment (zigzagcai) [reply .github/workflows/e2eaiok_nightly_pypi.yml]: For 1.0.0 release, it works well:
root@testserver:~/e2eAIOK# release_version=1.0.0
root@testserver:~/e2eAIOK# next_release_version=$(echo ${release_version} | awk -F. -v OFS=. 'NF==1{print ++$NF}; NF>1{$NF=sprintf("%0*d", length($NF), ($NF+1)); print}')
root@testserver:~/e2eAIOK# echo $next_release_version
1.0.1

**Review by xuechendi — APPROVED** [review]
LGTM

**Pull request merged**